src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotNetHelper.cs

```diff
@@ -126,7 +126,7 @@ void configureProcess(Process process, string? workingDirectory)
         }
     }
 
-    public static void ConfigureProcess(Process process, string? workingDirectory, bool setPath = false)
+    public static void ConfigureProcess(Process process, string? workingDirectory, bool setPath = true)
```

Comment:
I don't see a need to continue having the setPath parameter.  It is code bloat now that it is only being set to true.

Reply:
OK - will remove it and update the code.

Reply:
Fixed with https://github.com/dotnet/installer/pull/17238/commits/6da5d75d4775b12d2f56ce7fedba42c54b45ff21